Sources/SwiftUIKit/Views/Map.swift

```diff
@@ -0,0 +1,453 @@
+//
+//  Map.swift
+//  SwiftUIKit
+//
+//  Created by Oskar on 12/04/2020.
+//
+
+import Foundation
+import MapKit
+
+public class Map: MKMapView {
+  
+  fileprivate var initialCoordinates: CLLocationCoordinate2D
+  
+  fileprivate lazy var onFinishLoadingHandler: ((MKMapView) -> ())? = nil
+  
+  fileprivate lazy var afterRegionChangeHandler: ((MKMapView) -> ())? = nil
+  
+  fileprivate lazy var beforeRegionChangeHandler: ((MKMapView) -> ())? = nil
+  
+  fileprivate lazy var annotationViewConfigurationHandler: ((MKAnnotationView?, MKAnnotation) -> (MKAnnotationView?))? = nil
+  
+  fileprivate lazy var onAccessoryTapHandler: ((MKMapView, MKAnnotationView, UIControl) -> ())? = nil
+  
+  fileprivate lazy var onAnnotationViewStateChangeHandler: ((MKMapView, MKAnnotationView, MKAnnotationView.DragState, MKAnnotationView.DragState) -> ())? = nil
+  
+  fileprivate lazy var onAnnotationSelectHandler: ((MKMapView, MKAnnotationView) -> ())? = nil
+  
+  fileprivate lazy var onAnnotationDeselectHandler: ((MKMapView, MKAnnotationView) -> ())? = nil
+  
+  fileprivate lazy var annotationViewIdentifier: String? = nil
+  
+  public init(lat latitude: Double,
+              lon longitude: Double,
+              annotations: (() -> [(latitude: CLLocationDegrees,
```

Comment:
I'm thinking we might want to make this easier on the end user.

Could we make a `MapPoint` struct that contains these values? 

```
(latitude: CLLocationDegrees,
                                     longitude: CLLocationDegrees,
                                     title: String?,
                                     subtitle: String?)
```
to
```
public struct MapPoint {
    let latitude: Double
    let longitude: Double
    let title: String
    let subtitle: String
}
```

Reply:
Great idea! [Implemented](https://github.com/0xLeif/SwiftUIKit/pull/132/commits/ec9e95db49c665e96db7d7ce1e61849da7b42921).